plugins/acss-kit/scripts/detect_target.py

```
@@ -127,9 +148,190 @@ def main() -> int:
+        return self_test()
+
+    target = "react"
+    positional: list[str] = []
+    for a in args:
+        if a.startswith("--target="):
+            target = a.split("=", 1)[1].strip().lower()
+        else:
+            positional.append(a)
+
+    if target not in ("react", "html"):
+        print(f"Usage: detect_target.py [--target=react|html] [project_root]", file=sys.stderr)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)